packages/synthetix-main/contracts/modules/core/PoolConfigurationModule.sol

```diff
@@ -14,10 +14,10 @@ contract PoolConfigurationModule is IPoolConfigurationModule {
 
     using Pool for Pool.Data;
 
-    error PoolNotFound(uint128 poolId);
-
-    function setPreferredPool(uint128 poolId) external override poolExists(poolId) {
+    function setPreferredPool(uint128 poolId) external override {
         OwnableStorage.onlyOwner();
+        Pool.poolExists(poolId);
```

Comment:
can we just have it be `Pool.exists(poolId)` rather than `poolExists`?

Reply:
Renamed, but to `Pool.requireExists(poolId)` however, since `Pool.exists(poolId)` already exists =)

Reply:
awesome